translations/bnpchs-final-2018-blusukan.html

```diff
@@ -0,0 +1,95 @@
+<!-- Title: Blusukan -->
+<h3>Description</h3>
+
+<p style="text-align: justify;">
+  Bollyjee presidential election is around the corner. Jelix as the presidential 
+  candidate wants to realize the national commitments by doing <i>blusukan</i> (blusukan 
+  is indonesian word that refer to hands-on interaction with public in order to notice their 
+  necessity).
+</p>
+
+<p style="text-align: justify;">
+  There are a bunch of islands and $N$ province in Bollyjee. The island of two provinces lie on could be the 
+  same or different. Jelix had $M$ information, each information provide two provinces 
+  that lie on same island. If province $A$ and $B$ lie on the same island and province $B$ and $C$ lie 
+  on the same island then province $A$ and $C$ also lie on the same island.
+</p>
+
+<p style="text-align: justify;">
+  Each pair of province in the island is connected by a road. Also for each pair of road 
```

Comment:
```suggestion
  Each pair of provinces on an island are connected by a road. It is also known that for each pair of roads 
```

"Also" di awal kalimat itu struktur bahasa Indo